index.html

```diff
@@ -4,8 +4,66 @@
     <meta charset="utf-8">
     <title>Position Layout</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
   </head>
   <body>
+    <main class="section">
+      <img src="http://cdn.escapistmagazine.com/media/global/images/library/deriv/1000/1000267.jpg" alt="" class="section--bg-pic">
+      <div class="char-stats">
+        <img src="https://hydra-media.cursecdn.com/overwatch.gamepedia.com/thumb/4/4e/D.Va_portrait.png/100px-D.Va_portrait.png?version=f0ee344df879314523cecf83faa2e1ef" alt="" class="char-stats__img">
+        <div class="char-stats__health">
+          <p class="char-stats__num">
+            <strong class="char-stats__num2">500</strong>
+             /500
+          </p>
+          <p class="char-stats__hb">
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+            <span class="char-stats__hb2"></span>
+          </p>
+        </div>
+      </div>
+      <div class="middle-signs">
+        <div class="red-skull">
+          <img src="http://i.imgur.com/GMisBvu.png" alt="" class="red-skull-img">
+        </div>
 
+        <div class="eliminate1">
+          <p class="eliminate1-font">
+            <strong class="eliminate1-white">ELIMINATED</strong> REAPER
+          </p>
+        </div>
+
+        <div class="eliminate2">
+          <p class="eliminate2-font">
+            <strong class="eliminate2-white">ELIMINATED</strong> REAPER
+          </p>
+        </div>
+        <div class="weapon-stats">
```

Comment:
Weapons stats right now is inside of `middle-signs` if you close the `middle-signs` div before weapon stats, it will be easier to move around the eliminations and red skull together instead of as separate position fixed elements.